### PR TITLE
HTTP header in MDC test

### PR DESCRIPTION
**Pull request by meltsufin**
Run `FrontEnd` and then use curl to invoke:

```
curl http://localhost:8081/ -H 'req-id: blah'
```

Output:
```
{traceId=5e7cfe9e7c937391754fae51e18d98ed, spanId=754fae51e18d98ed, spanExportable=true, req-id=blah, X-Span-Export=true, X-B3-SpanId=754fae51e18d98ed, X-B3-TraceId=5e7cfe9e7c937391754fae51e18d98ed}
```

### CODE REVIEW

**Comment by meltsufin** [comment]
Sorry, didn't mean to create the PR here.

File: src/main/java/sleuth/webmvc/Frontend.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package sleuth.webmvc;
 
+import org.jboss.logging.MDC;
```
Comment (codefromthecrypt) [inline src/main/java/sleuth/webmvc/Frontend.java:3]: hopefuly this is the wrong MDC :D

**Comment by codefromthecrypt** [comment]
actually you could make a PR if you want to use "user-name" like we have in the brave example https://github.com/openzipkin/brave-webmvc-example/blob/master/webmvc4/src/main/java/brave/webmvc/TracingConfiguration.java#L53

**Comment by codefromthecrypt** [comment]
https://github.com/openzipkin/brave-webmvc-example/blob/master/webmvc4/src/main/java/brave/webmvc/Backend.java#L15-L21

**Comment by codefromthecrypt** [comment]
just forgot to update here is all..

File: src/main/java/sleuth/webmvc/Frontend.java (hunk 1 above)
Comment (meltsufin) [reply src/main/java/sleuth/webmvc/Frontend.java:3]: Oops, still works though. 😊 
Looks like it's brought in by Hibernate Validator.

**Pull request closed without merging**